src/components/stepTwo/StepTwoForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import { observer } from 'mobx-react'
+import { FormSpy } from 'react-final-form'
+import { TokenName } from '../Common/TokenName'
+import { TokenTicker } from '../Common/TokenTicker'
+import { TokenDecimals } from '../Common/TokenDecimals'
+
+const errorStyle = {
+  color: 'red',
+  fontWeight: 'bold',
+  fontSize: '12px',
+  width: '100%',
+  height: '10px',
+}
+
+@observer
```

Comment:
Does this need to be an observer?

Reply:
It wasn't necessary. Converted it to a functional component.